packages/styles/tooltip.css

```diff
@@ -124,15 +124,14 @@
 
 .TooltipHead {
   background: var(--gray-20);
-  font-size: 16px;
+  font-size: var(--text-size-small);
   color: var(--gray-90);
-  padding: 16px 28px;
+  padding: var(--space-small) var(--space-large);
   text-align: center;
 }
 
 .TooltipContent {
   background: #fff;
-  font-size: var(--font-size-smaller);
```

Comment:
Seems like it was a typo and should have been `--text-size-smaller`.

PR that added this CSS:
https://github.com/dequelabs/cauldron/pull/134/files

Reply:
Good point, however the parent class `.Tooltip` has `font-size: --text-size-smaller`, so it is inherited from there:
https://github.com/dequelabs/cauldron/blob/60782888daab263a83724252e8891c1317da2c89/packages/styles/tooltip.css#L22-L27

![Screen Shot 2022-10-19 at 9 42 20 AM](https://user-images.githubusercontent.com/31857969/196753206-f7c16f8a-f8d8-474b-9e7c-2b4ae441b458.png)

I decided to remove it because doing so had no change on the appearance. And assuming that it was a typo, the presumably desired font size is applied. Does that make sense or do you think correcting instead of removing would be better?